phpBB/styles/prosilver/theme/print.css

```diff
@@ -137,3 +137,10 @@ div.spacer { clear: both; }
 .skip_link { display: none; }
 
 .codebox p { display: none; }
+
+.emoji {
```

Comment:
this needs to be ignored by linting so wrap the class like so pls

/* stylelint-disable declaration-property-unit-whitelist */
.emoji {
	min-height: 18px;
 	min-width: 18px;
 	height: 1em;
 	width: 1em;
 }
/* stylelint-enable declaration-property-unit-whitelist */